scss/docs/_patterns_icons.scss

```diff
@@ -44,5 +44,98 @@
 
 // These are not included in Vanilla by default,
 // so include them here for the purposes of
-// presenting the application icons on the docs page.
-@include vf-p-icons--application;
+// presenting the additional icons on the docs page.
+@include vf-p-icon-applications;
```

Comment:
I'm not sure if we should do it here. While adding couple "application" icons wasn't a huge deal, now we are adding 100 icons to docs CSS, while there are needed on one single page.

Maybe it would be worth trying just adding a separate icons CSS to the icons page of the docs? We already have a standalone CSS with just the icons for the example, so we could reuse it and avoid adding all the icons to the docs CSS. Especially that we wanted to get rid of custom docs styling ideally.

Reply:
In that case - is there a way we can include CSS files on a specific doc page? In this case we're talking about a markdown doc, and I don't think there's front matter available for this purpose. I could drop a style tag in at the top of the document but that's a little dirty.

Reply:
Could we add it to `context` in frontmatter maybe?

```
context:
   title: Icons
   stylesheet: /path/to/styles.css
```

And then in `_root.html` layout check for `{{ stylesheet }}` and add it if needed?
I just quickly checked and it seems we can add custom stuff to `context`.